python/graphstorm/dataloading/dataloading.py

```diff
@@ -278,6 +285,12 @@ def __next__(self):
 
         return (input_nodes, batch_graph, blocks)
 
+    def __len__(self):
+        # Follow
+        # https://github.com/dmlc/dgl/blob/1.0.x/python/dgl/distributed/dist_dataloader.py#L116
+        # DistDataLoader.expected_idxs is the length of the datalaoder
+        return self.dataloader.expected_idxs
```

Comment:
is this the number of nodes/edges or the number of batches?

Reply:
The number of batches

Reply:
i see. this kind of ties our implementation with DGL's internal implementation.